daiquiri/metadata/handlers.py

```diff
@@ -8,33 +8,52 @@
 
 @receiver(post_save, sender=Schema)
 def schema_updated_handler(sender, **kwargs):
-    if not kwargs.get('raw') and apps.is_installed('daiquiri.oai'):
-        from daiquiri.oai.utils import update_records
-        update_records('schema', kwargs['instance'])
+    if not kwargs.get('raw'):
+        if apps.is_installed('daiquiri.oai'):
+            from daiquiri.oai.utils import update_records
```

Comment:
this is the cost of putting the adapter into the corresponding app, right? i.e: inline imports.